source/common/redis/command_splitter_impl.h

```diff
@@ -118,6 +118,21 @@ struct InstanceStats {
   ALL_COMMAND_SPLITTER_STATS(GENERATE_COUNTER_STRUCT)
 };
 
+/**
+ * Struct definition for supported commands and how to handle them
```

Comment:
i'd remove struct definition part, as it's obvious :)

"Supported commands and how to handle them." should be enough.